### PR TITLE
[228] - [BugTask] Uploaded files or pictures can still be uploaded again

### PR DESCRIPTION
**Pull request by ww-jermaine**
## Issue Link
- https://app.asana.com/0/1203011167276287/1203290388147228/f

## Definition of Done
- [x] Fixed the issue with file reuploads

## Notes
- None

## Expected Output
- If the file is already existing, the app will add suffix. 
 

## Screenshots/Recordings

https://user-images.githubusercontent.com/109291819/199903569-381295c5-e822-499e-8cc7-069509b12139.mp4

### CODE REVIEW

**Review by jsvelte — APPROVED** [review]
# Approved 💯

File: api/app/Http/Controllers/ProjectFileController.php [hunk 1]
```diff
@@ -28,11 +28,36 @@ public function index(Project $project)
...
+            //if the count is greater than 1, then update the most recent file and add suffix
+            // if ($count > 0) {
+            //     $most_recent_media = $project->getMedia('project-files')->sortByDesc('created_at')->last();
+            //     // rename the $most_recent_media by adding a number to the end of the file name
+            //     //$most_recent_media->file_name = $most_recent_media->file_name . ' (' . $count . ')';
+            //     $most_recent_media->update([
+            //         'name' => $most_recent_media->name . ' (' . $count . ')',
+            //         'file_name' => $most_recent_media->name . ' (' . $count . ')' . '.' . $most_recent_media->extension
+            //     ]);
+            // }
```
Comment (abduljalilpalala) [inline api/app/Http/Controllers/ProjectFileController.php]: Remove comment if not needed.

**Review by abduljalilpalala — APPROVED** [review]
We can still improve the implementation on the ProjectFileController.php file but for now I approve this PR. I think it's best to keep the controllers with only `store`, `show`, `index`, `update` and `destroy` methods and if there are additional methods to be used then we can separate those methods in a util, service or helper files.

**Review by jsvelte — APPROVED** [review]
# Approved

**Review by rogeliojohnoliverio — APPROVED** [review]
Approved 🙇

**Comment by ww-jermaine** [comment]
> We can still improve the implementation on the ProjectFileController.php file but for now I approve this PR. I think it's best to keep the controllers with only `store`, `show`, `index`, `update` and `destroy` methods and if there are additional methods to be used then we can separate those methods in a util, service or helper files.

I thought these are just boilerplate codes that laravel produces using artisan and controllers can have as many methods as they need. https://stackoverflow.com/questions/39797502/should-all-controllers-have-only-the-basic-crud-methods

**Pull request merged**